src/evo/providertx.h

```diff
@@ -74,6 +79,13 @@ class CProRegTx
                 obj.scriptPayout,
                 obj.inputsHash
         );
+        if (obj.nVersion == BASIC_BLS_VERSION && obj.nType == TYPE_HIGH_PERFORMANCE_MASTERNODE) {
```

Comment:
why bls check? oh, because this "version" also enables HPMNs, maybe that should be communicated in the version?

Reply:
@PastaPastaPasta Yes, that's why. 
What do you mean by _communicated in the version_ ?

Reply:
something like `BASIC_BLS_VERSION` -> `BASIC_BLS_HPMN_COMPITABLE_VERSION` but that's a little verbose so idk 🤷

Reply:
That's verbose I agree 🤷